Makefile

```diff
@@ -26,6 +26,11 @@ SLOW_SPEC_THRESHOLD := 120
 # To enable verbosity export or set env GINKGO_VERBOSE_MODE like "GINKGO_VERBOSE_MODE=-v"
 GINKGO_VERBOSE_MODE ?=
 
+# Env variable UNIT_TEST_VERBOSE_MODE is used to get control over enabling go test
+# verbose mode for unit test run. By default go test verbosity is not enabled.
+# To enable verbosity export or set env UNIT_TEST_VERBOSE_MODE like "UNIT_TEST_VERBOSE_MODE=-v"
+UNIT_TEST_VERBOSE_MODE ?=
```

Comment:
it should either be `UNIT_TEST_VERBOSE_MODE=true` or `UNIT_TEST_ARGS=-v`

Reply:
@girishramnani 
> UNIT_TEST_VERBOSE_MODE=true

Looks good, but it needs few more unnecessary lines in make file to implement this. Unnecessary because we have already a ```-v``` flag which enables unit test verbose in golang. For example ```go test -v```. Here just need to use the ```-v``` flag through an env var.

> UNIT_TEST_ARGS=-v

Sounds good, but it is not specific around which args it holds. So imo ```UNIT_TEST_VERBOSE_MODE ?=``` is not a bad choice ;)

Reply:
> > UNIT_TEST_ARGS=-v
> 
> Sounds good, but it is not specific around which args it holds. So imo `UNIT_TEST_VERBOSE_MODE ?=` is not a bad choice ;)

That is exactly the intention. This variable can hold any flags for `go test` not just `-v`. So something like `UNIT_TEST_ARGS` makes more sense. 
If you want to limit it to just verbose mode than you should make it `UNIT_TEST_VERBOSE_MODE=true` and add those few additional lines.


Btw `GINKGO_VERBOSE_MODE` has the same issue ;-)

Reply:
couldn't have said it better myself :+1: @kadel

Reply:
@kadel Awesome explanation. Review comments are addressed.